examples/java/pom.xml

```diff
@@ -48,6 +48,12 @@
             <artifactId>selenium-logger</artifactId>
             <version>2.4.0</version>
         </dependency>
+    
+      <dependency>
+        <groupId>org.scala-lang</groupId>
```

Comment:
is this dep required ?

Reply:
I'll go through the dependencies and see if an array/list library is already being used in Java

Reply:
No it's not - it was erroneously added by vscode. I'm using java.util.List.

I'll remove it

Reply:
Thank you!